contract-message-receiver/src/contract_message_receiver_abi.sw

```diff
@@ -0,0 +1,6 @@
+library contract_message_receiver_abi;
```

Comment:
I'd refer to the generic `interface` rather than `abi`

Reply:
I'll let everyone else chime in, but I believe I used `abi` because I saw it in other sway projects